src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,17 +243,29 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
 
             private void AddExceptions(ISymbol symbol)
             {
-                var exceptionTypes = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true).ExceptionTypes;
+                const int maxTextSize = 70;
```

Comment:
where did this value come from?

Reply:
It came from above when someone suggested the max size before truncation per exception documentation should be 70, however someone else things that there perhaps should be no truncation and that all of the possible truncation / limiting what exceptions the IDE shows during pagination should be up to the IDE team.